models/contacts.ts

```diff
@@ -0,0 +1,13 @@
+interface Contacts {
```

Comment:
This seems great to define a structure, though would we want to implement a solution that could be more general as to account for contact methods we didn't think of?

Reply:
I guess we could have an opportunity to upload a link?